include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -41,6 +41,9 @@ class CommonOperatorLoader : public ProtobufLoader {
 
   virtual bool getBroadcast(const ArgumentDictionaryTy &dict) { return true; }
 
+  /// Inform is a given operator supports Multidirectional broadcasting
```

Comment:
Period.

Reply:
I don't understand the semantics of this method. Can you rewrite the comment as:

/// \returns True if the operator with the name \p typeName has support for XXX and YYY.

Reply:
Please use llvm::StringRef for the parameter.